doc/source/examples/diffraction_objects_example.rst

```diff
@@ -117,6 +117,13 @@ For convenience, you can also apply an offset to the scaled new diffraction obje
 
     scaled_and_offset_measured = measured.scale_to(calculated, q=5.5, offset=0.5)
 
+You can call `scale_to()` without specifying a value for `q`, `tth`, or `d`.
```

Comment:
This will be the default behavior so should go first in the docs.    Then we can say something like "if this doesnlt give desirable results you can user....." and describe the other workflow.

To give even more flexibility to the user we may want to allow them to specify a separate xtype and xvalue for the target and the scaled DO.  The hierarchy woiuld be:
1. no xtype/xvalue
2. if that doesn't give desirable results, specify xtype and xvalue for target.  scale to the same (nearest) point in the scaled pattern
3. if that doesn't give desirable results, specify xtype, and a different xvalue for target. and scaled DO, respectively.  Scale the yvale of xvalue_scaled to yvalue of xvalue_target.

This last functionality is a new one and we should put it on a new issue (and a new PR if we want to implement it)